docs/fides/docs/language/resources/organization.md

```diff
@@ -11,7 +11,7 @@ The fides key for the default Organization is `default_organization`
 
 **fides_key**<span class="required"/>&nbsp;&nbsp;_string_
 
-A string token of your own invention that uniquely identifies this Organization. It's your responsibility to ensure that the value is unique across all of your Organization objects. The value should only contain alphanumeric characters and underbars (`[A-Za-z0-9_]`). 
+A string token of your own invention that uniquely identifies this Organization. It's your responsibility to ensure that the value is unique across all of your Organization objects. The value should only contain alphanumeric characters and underbars (`[A-Za-z0-9_.-]`). 
```

Comment:
```suggestion
A custom string token that uniquely identifies this Organization. It must be unique across all Organization objects. The value may only contain alphanumeric characters, `_`, `.`, and `-` (it will be evaluated against the regular expression: `[A-z0-9_.-]`).
```